### PR TITLE
Use zeroMemory over HEAPU8.fill. NFC

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

**Comment by kripken** [comment]
> Also, add missing return statement to the legacy path in zeroMemory.

I don't see that?

File: src/library_webgpu.js [hunk 1]
```diff
@@ -2052,7 +2052,7 @@ var LibraryWebGPU = {
     }
 
     var data = _memalign(16, mapped.byteLength);
-    HEAPU8.fill(0, data, mapped.byteLength);
+    zeroMemory(data, mapped.byteLength);
```
Comment (kripken) [inline src/library_webgpu.js:2055]: This does not look NFC, as before we did
```js
HEAPU8.fill(0, data, mapped.byteLength);
```
while after we are effectively doing
```js
HEAPU8.fill(0, data, data + mapped.byteLength);
```
Perhaps it was wrong before?

Comment (kainino0x) [reply src/library_webgpu.js:2055]: Oops, it was definitely wrong before.

**Review by kainino0x — APPROVED** [review]
LGTM, but same comment about the return statement.

**Comment by sbc100** [comment]
> > Also, add missing return statement to the legacy path in zeroMemory.
> 
> I don't see that?

Reverted

**Pull request merged**